### ISSUE TITLE
Content.Load<Song>() tries to load OGG on Windows project

### ISSUE THREAD

**Issue by WickedPeanuts** [MonoGame/MonoGame#7329]
Hello Monogame community. First of all, I don't know exacly if it is a bug or not. For me it looks more like the way VS compiler handles and builds the assemblies or maybe I have done something wrong. For this, I have decided to write this issue to warn the others about this problem and even learn something new.

I've reproduced it on the current build (3.8.0.1641) and haven't tested the same thing on previous ones.

My solution is designed like this:

Project 1 - Game Project <- All contents are here. Content.Load<Song>() is also called in this project.
Project 2 - Library Project

Where game contains a reference for the library project and both contains a reference for Monogame assemblies. Initially I was testing my game's performance on DesktopGL and then I have decided to compare it on WindowsDX.
I have updated the Game Project reference to WindowsDX, but I forgot to update the Library's reference to WindowsDX. And that was my mistake. For my surprise the Game Project's Content.Load<Song> was using it's DesktopGL version and was attempting to open songs as OGG instead WindowsDX's WMA.

I believe the exception is well know here: System.IO.InvalidDataException: 'Could not determine container type!'.

#### What version of MonoGame does the bug occur on:
- MonoGame 3.8.0.1641

#### What operating system are you using:
- Windows

#### What MonoGame platform are you using:

- WindowsDX

**Comment by harry-cpp** [comment]
If you can, please create a minimal project to reproduce the problem.

**Comment by WickedPeanuts** [comment]
Thanks for the quick reply! 
I have created this rep with the problem.

https://github.com/WickedPeanuts/Monogame-Issue7329

**Comment by NicusorN5** [comment]
For playing a song, SoundEffect is good too. Just make sure that your MGCB settings are correct. (Just a workaround)

**Comment by WickedPeanuts** [comment]
Well, in this case the sound effects were working correctly and the MGCB was set to windows (dx) as well.

**Comment by harry-cpp** [comment]
I see what your problem is, you incorrectly setup your class library, change:
```
<ItemGroup>
  <PackageReference Include="MonoGame.Framework.DesktopGL" Version="3.8.0.1641" />
</ItemGroup>
```

to:
```
<ItemGroup>
  <PackageReference Include="MonoGame.Framework.DesktopGL" Version="3.8.0.1641">
    <PrivateAssets>All</PrivateAssets>
  </PackageReference>
</ItemGroup>
```

(also you forgot to initialize c1 and forgot to add a Program.cs)

**Comment by WickedPeanuts** [comment]
Thanks for the reply once again,

Game Project (The project that calls Content.Load) includes the right WinDX version
>   <ItemGroup>
>     <PackageReference Include="MonoGame.Framework.WindowsDX" Version="3.8.0.1641" />
>     <PackageReference Include="MonoGame.Content.Builder.Task" Version="3.8.0.1641" />
>   </ItemGroup>

As I stated before, I did purposely imported the wrong version on the lib project, but the game project has the right one. The reason I created this (possible) issue is because I do not understand the reason why GameProject decided to use LibClass DesktopGL instead its own WindowsDX.

Well, and about the library project, it has no program.cs, yeah. And the instance thing was my mistake, but the code wouldn't reach that far since it crashes on LoadContent.
